BuildingSync.xsd

```diff
@@ -13934,6 +13879,67 @@
       <xs:element name="Other" type="auc:OtherType" minOccurs="0"/>
     </xs:choice>
   </xs:complexType>
+  <xs:complexType name="SpatialUnitTypeType">
+    <xs:sequence>
+      <xs:element name="SpatialUnitType" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Unit type within the premises.</xs:documentation>
+        </xs:annotation>
+        <xs:simpleType>
+          <xs:restriction base="xs:string">
+            <xs:enumeration value="Lots"/>
+            <xs:enumeration value="Parking spaces"/>
+            <xs:enumeration value="Apartment units"/>
+            <xs:enumeration value="Businesses"/>
+            <xs:enumeration value="Guest rooms"/>
+            <xs:enumeration value="Stations"/>
+            <xs:enumeration value="Buildings"/>
+            <xs:enumeration value="Areas"/>
+            <xs:enumeration value="Thermal Zones"/>
+            <xs:enumeration value="Floors"/>
+            <xs:enumeration value="Rooms"/>
+            <xs:enumeration value="Bedrooms"/>
+            <xs:enumeration value="Other"/>
+            <xs:enumeration value="Unknown"/>
+          </xs:restriction>
+        </xs:simpleType>
+      </xs:element>
+      <xs:element name="NumberOfUnits" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Number of individual units within the premises.</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+          <xs:simpleContent>
+            <xs:extension base="xs:integer">
```

Comment:
breaking change, but this should be a `postiveInteger`. Something to address in v3? Created ticket here: #301

Reply:
true. I think I will leave the current implementation to remain non-breaking, but started adding checklists to #301